core/designsystem/src/main/java/co/kr/tnt/designsystem/component/Button.kt

```diff
@@ -0,0 +1,202 @@
+package co.kr.tnt.designsystem.component
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.kr.tnt.core.designsystem.R
+import co.kr.tnt.designsystem.theme.TnTTheme
+import co.kr.tnt.designsystem.util.ButtonSize
+import co.kr.tnt.designsystem.util.ButtonType
+import co.kr.tnt.designsystem.util.IconPosition
+import co.kr.tnt.designsystem.util.getButtonConfig
+
+@Composable
+fun TnTTextButton(
+    size: ButtonSize = ButtonSize.Medium,
+    type: ButtonType = ButtonType.Primary,
+    text: String,
+    enabled: Boolean = true,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val config = getButtonConfig(size, type, enabled)
+
+    Button(
+        onClick = onClick,
+        enabled = enabled,
+        colors = config.colors,
+        shape = RoundedCornerShape(config.cornerRadius),
+        border = BorderStroke(width = config.stroke, color = config.borderColor),
```

Comment:
```suggestion
        border = BorderStroke(
            width = config.stroke,
            color = config.borderColor,
        ),
```

취향차이긴 한데 파라미터 두 개이상인 경우 개행해주는거 어떨까요 ?.?

Reply:
옹 코드가 길어지는 것 때문에 개행하지 않고 구현했습니다
개행하는거 좋아요~! 수정해두겠습니다